### PR TITLE
Navigation: Use page nav in silence and policy notification pages

### LINKED ISSUE #55110: Page layout: Alerting -> Notification policies -> Add mute timing
- [ ] convert to use `navId` instead of `navModel`
- [ ] add `subTitle`
- [ ] add `pageNav` (see `TeamPages`)

### LINKED ISSUE #55111: Page layout: Alerting -> Silences -> New silence
- [ ] convert to use `navId` instead of `navModel`
- [ ] add `subTitle`
- [ ] add `pageNav` (see `TeamPages`)

### PR DESCRIPTION
**Pull request by lpskdl**
**What this PR does / why we need it**:
- added pageNav for alert silence pages (edit/new)

**with top nav**
Screenshot 2022-09-26 at 16 40 09
Screenshot 2022-09-26 at 16 39 43

**with top nav off**
Screenshot 2022-09-27 at 10 09 19
Screenshot 2022-09-27 at 10 09 12

**Which issue(s) this PR fixes**:

Fixes #55110
Fixes #55111 

**Special notes for your reviewer**:
- This is my initial attempt to update the pageNav in alerting silence area. I think this is the least change I can do to include the pageNav. I did try to move each silence pages(list, add, edit) to distribute the responsibilities and reduce the complexity of `Silence.tsx` page it looks ok but I hesitated to continue doing it as there might be more impacted areas with that approach.

### CODE REVIEW

File: public/app/features/alerting/unified/hooks/useSilenceNavData.ts
Comment (torkelo) [inline public/app/features/alerting/unified/hooks/useSilenceNavData.ts:21]: this is missing a breadcrumbs bit (so that it get's a breadcrumb when topnav is disabled)

Comment (lpskdl) [reply public/app/features/alerting/unified/hooks/useSilenceNavData.ts:26]: thanks @torkelo , I added the missing breadcrumbs

**Review by ashharrison90 — APPROVED** [review]
looks great! again, should wait for @grafana/alerting-squad-frontend review before merging 🙂

**Comment by lpskdl** [comment]
HI @grafana/alerting-squad-frontend , I would like to ask if you could take a look in the changes related to silence and policy notification pages to support the new navigation.

The changes can be differentiate by adding/removing **topnav** in feature_toggles settings(custom.ini or defaults.ini)

Thanks a lot

**Comment by lpskdl** [comment]
> HI @grafana/alerting-squad-frontend , I would like to ask if you could take a look in the changes related to silence and policy notification pages to support the new navigation.
> 
> The changes can be differentiate by adding/removing **topnav** in feature_toggles settings(custom.ini or defaults.ini)
> 
> Thanks a lot

Hi @grafana/alerting-squad-frontend, I am going to merge this in. If there are some later comments I am going to address it in another PR

**Pull request merged**
